stdlib/public/SDK/Foundation/NSError.swift

```diff
@@ -389,11 +389,11 @@ public protocol _BridgedStoredNSError :
 }
 
 /// TODO: Better way to do this?
-internal func _stringDictToNSObjectDict(_ input: [String : Any])
-    -> [NSObject : Any] {
-  var result: [NSObject : Any] = [:]
+internal func _stringDictToAnyHashableDict(_ input: [String : Any])
+    -> [AnyHashable : Any] {
+  var result: [AnyHashable : Any] = [:]
```

Comment:
While you are changing this code, could you reserve capacity in the new dictionary?